nullaway/src/main/java/com/uber/nullaway/GenericsChecks.java

```diff
@@ -286,4 +325,66 @@ private Type.ClassType typeWithPreservedAnnotations(ParameterizedTypeTree tree)
             type.getEnclosingType(), com.sun.tools.javac.util.List.from(newTypeArgs), type.tsym);
     return finalType;
   }
+
+  public void checkTypeParameterNullnessForAssignabilityForConditionalExpression(
+      ConditionalExpressionTree tree) {
+    if (!config.isJSpecifyMode()) {
+      return;
+    }
+    //  Tree lhsTree;
+    Type lhsType;
+    Tree truePartTree;
+    Tree falsePartTree;
+
+    lhsType = ASTHelpers.getType(tree);
+    truePartTree = tree.getTrueExpression();
+    falsePartTree = tree.getFalseExpression();
+    // rhsTree can be null for a VariableTree.  Also, we don't need to do a check
+    // if rhsTree is the null literal
+    if (truePartTree == null
+        || truePartTree.getKind().equals(Tree.Kind.NULL_LITERAL)
+        || falsePartTree == null
+        || falsePartTree.getKind().equals(Tree.Kind.NULL_LITERAL)) {
+      return;
+    }
+    Type truePartType = ASTHelpers.getType(truePartTree);
```

Comment:
Again, we have duplicated code.  This is not exactly the same as the other cases, but let's try to re-use the extracted function I suggested above as much as possible (there will be minor differences here in the code before the call).